kubernetes/deploy.sh

```diff
@@ -2,23 +2,51 @@
 
 # Deploys CoreDNS to a cluster currently running Kube-DNS.
 
-SERVICE_CIDR=$1
-POD_CIDR=$2
-CLUSTER_DNS_IP=$3
-CLUSTER_DOMAIN=${4:-cluster.local}
-YAML_TEMPLATE=${5:-`pwd`/coredns.yaml.sed}
-
-if [[ -z $SERVICE_CIDR ]]; then
-	echo "Usage: $0 SERVICE-CIDR [ POD-CIDR ] [ DNS-IP ] [ CLUSTER-DOMAIN ] [ YAML-TEMPLATE ]"
-	exit 1
-fi
+show_help () {
+cat << USAGE
+usage: $0 [ -r REVERSE-CIDR ] [ -i DNS-IP ] [ -d CLUSTER-DOMAIN ] [ -t YAML-TEMPLATE ]
+
+    -r : Define a reverse zone for the given CIDR. You may specifcy this option more
```

Comment:
There is a sample of deploy.sh in the README.md.
Need to update I think.

Reply:
thanks - updated readme